CODEOWNERS

```diff
@@ -48,7 +48,7 @@
 
 /Workbooks/Traffic*Analytics/** @microsoft/network-watcher-traffic-analytics @microsoft/applicationinsights-devtools
  
-/Workbooks/SapMonitor/** @microsoft/azuremonitorforsapsolutions @microsoft/applicationinsights-devtools
+/Workbooks/SapMonitor/** @microsoft/sap-embrace-idc-monitor @microsoft/azuremonitorforsapsolutions @microsoft/applicationinsights-devtools
```

Comment:
i believe there's at least one other line you want to touch here?

`/gallery/workbook/Microsoft.HanaOnAzure-sapMonitors.json @microsoft/azuremonitorforsapsolutions`

other option: you could add the new users to this existing team instead of adding another team?
then you wouldn't have to change the file at all?

if you are going to use this new team i'll have to give write access to this team as well

Reply:
Updated CODEOWNER for `/gallery/workbook/Microsoft.HanaOnAzure-sapMonitors.json @microsoft/azuremonitorforsapsolutions` as well